DependencyInjection/Configuration.php

```diff
@@ -179,9 +179,33 @@ private function addErrorHandlerSection(NodeBuilder $nodeBuilder)
     {
         $nodeBuilder
             ->arrayNode('error')
+                ->fixXmlConfig('template')
+                ->fixXmlConfig('exclusion_rule')
                 ->children()
                     ->scalarNode('enable_parent_provider')->defaultValue(false)->end()
                     ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
+                    ->arrayNode('templates')
+                        ->useAttributeAsKey('format')
+                        ->requiresAtLeastOneElement()
+                        ->defaultValue(array(
+                            'html' => 'CmfSeoBundle:Exception:exception.html.twig',
+                            'xml'  => 'TwigBundle:Exception:error.xml.twig',
+                            'json' => 'TwigBundle:Exception:error.json.twig',
```

Comment:
should we remove xml and json here and just fall through to the default error controller, now that we fall through on not found templates?